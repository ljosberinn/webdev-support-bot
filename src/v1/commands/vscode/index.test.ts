import handleVSCodeRequest from '.';

test('inform users that visual studio code is great and free', async () => {
  const sendMock = jest.fn();
  const msg: any = {
    channel: { send: sendMock },
    client: {
      emojis: {
        cache: [':vscode:'],
      },
    },
  };

  await handleVSCodeRequest(msg);
  expect(msg.channel.send.mock.calls[0][0]).toMatchSnapshot();
});
