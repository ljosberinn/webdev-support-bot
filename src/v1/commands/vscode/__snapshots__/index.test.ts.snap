// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inform users that visual studio code is great and free 1`] = `
"> :bulb: consider using a lightweight, customizeable and monthly updated editor such as
> 
> Visual Studio Code - <https://code.visualstudio.com/>
> 
> It's free & available cross platform and next to WebStorm the go to editor in this industry."
`;
